CHANGELOG.md

```diff
@@ -47,6 +47,9 @@ All notable changes to this project will be documented in this file.
     * #### Removed
 * ### NI-DCPower
     * #### Added
+        * `create_advanced_sequence()` - [#504](https://github.com/ni/nimi-python/issues/504)
```

Comment:
you also added a couple of attributes, delete_advanced_sequence...

Reply:
Added